### PR TITLE
Setting default reddit image

### PR DESCRIPTION
**Pull request by Jovialiste82**

### CODE REVIEW

**Comment by Jovialiste82** [comment]
Hey guys, just added some code so we can have a default thumbnail link whenever the Reddit API sends back a "default" or "self" value for image instead of a thumbnail link

File: server/seeder.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const mongoose = require("mongoose");
 const dotenv = require("dotenv");
 const users = require("./data/users");
-const mentions = require("./data/mentions");
+// const mentions = require("./data/mentions");
```
Comment (WhoaNellie) [inline server/seeder.js]: Can remove commented out code from this file

File: server/utils/reddit.js [hunk 2]
```diff
@@ -6,6 +6,14 @@ const searchRecursive = async (term, after = "", posts = []) => {
 
   let url = `${ENDPOINT}?q=${term}` + (after ? `&after=${after}` : "");
 
+  const image = (image) => {  
+    if(image === "default" || image === "self") {
+      return "https://i.imgflip.com/1f0hdj.jpg"
```
Comment (WhoaNellie) [inline server/utils/reddit.js]: Might consider downloading this and linking to it locally on the FE

**Review by WhoaNellie — COMMENTED** [review]
You might consider fixing this on the front end instead by reading if the link says "default" or "self" and putting in a locally imported image so you can cut down on external links.

**Comment by Jovialiste82** [comment]
@WhoaNellie  i just moved the logic from BE to FE

**Review by jessdesr — APPROVED** [review]

**Pull request merged**